src/components/ChooseOfferForm/ChooseOfferForm.tsx

```diff
@@ -81,7 +81,7 @@ const ChooseOfferForm: React.FC<Props> = ({
         aria-label={ariaLabel}
       />
       <label className={styles.label} htmlFor={offer.offerId}>
-        <h4 className={styles.offerTitle}>{title}</h4>
+        <h2 className={styles.offerTitle}>{title}</h2>
```

Comment:
I agree with this h2 change, but I noticed a parent <label> containing a whole lot of HTML. I think the label should be option role or something similar. Not a quickwin, but make sure you have wrote it down in your sheet because this should use some optimisation ;)